stdlib/2/ast.pyi

```diff
@@ -1,6 +1,6 @@
 # Python 2.7 ast
 
-import typing
+import typing as _typing
```

Comment:
Could you add a small comment explaining this problem here (and in the Python 3 stub) for future generations?

Reply:
Done.